styles/components/WorkSectionAsGallery.scss

```diff
@@ -38,28 +28,39 @@
     &--dark-gradient {
       @include media('md') {
         background: linear-gradient(
-          180deg,
-          transparent 80%,
-          rgba(0, 0, 0, 0.5) 130%
+          0deg,
+          rgba(0, 0, 0, 0.32) -3.5rem,
+          transparent 5.5rem
         );
       }
     }
 
-    @extend .tiny;
     line-height: 1.125rem;
+
+    font-family: $sans-serif-stack;
+    font-weight: 400;
+
     margin: 0;
+    font-size: 0.75rem;
+    letter-spacing: 0.4px;
 
     @include media('md') {
-      line-height: 1.5rem;
+      margin: 0 2px;
+      font-size: 0.9375rem;
+      letter-spacing: 0.4px;
     }
```

Comment:
i think we should keep using `@extend .tiny` here since our intention is to keep the caption close to that style, except the `line-height` and mobile `margin`

<img width="300" alt="Screen Shot 2023-02-14 at 10 08 17 AM" src="https://user-images.githubusercontent.com/30246503/218689758-4a9fd942-e16a-49da-80f5-710c96f1147e.png">

Reply:
@mokaymokay now .tiny has different style(screenshot attached, also it's on the branch's latest codebase!) since we reverted all type changes as per Hugh's request([twist thread](https://twist.com/a/133876/ch/552738/t/4052838/c/81830765)). The caption text in new design for WG([figma](https://www.figma.com/file/M6zHKnDZ7ALV5pZxPvVSHZ/sanctuary.computer-refresh?node-id=59%3A347&t=JjIMF7a139EhpRTL-1)) is not following the existing type style(ex. .tiny / .small), but we wanted to follow the design so I added separate manual type style in this file!
<img width="271" alt="Screen Shot 2023-02-14 at 8 57 18 AM" src="https://user-images.githubusercontent.com/12539032/218759894-fb70b1da-8255-4f38-a80a-3276184004e5.png">

Reply:
ok just extend whichever is closest and override! @nahbee10